etl/steps/data/garden/wb/2025-04-14/world_bank_pip_dimensional.meta.yml

```diff
@@ -0,0 +1,1223 @@
+        short_unit: $
+        description_short: The mean <<welfare_type>> per day within the {definitions.percentiles_avg_share} (tenth of the population).
+        description_key:
+          - "{definitions.description_key_ppp}"
+          - "{definitions.description_key_welfare_type}"
+          - "{definitions.description_key_not_market_income}"
+        description_processing: |-
+          {definitions.description_processing_avg}
+          {definitions.description_processing_main}
+        display:
+          name: Average ({definitions.percentiles_avg_share})
+          numDecimalPlaces: 2
+          <<: *common_display
+        presentation:
+          grapher_config:
+            title: Mean <<welfare_type>> per day within the {definitions.percentiles_avg_share}
+            subtitle: The mean <<welfare_type>> per day within the {definitions.percentiles_avg_share} (tenth of the population). {definitions.ppp_adjustment_subtitle}
+            note: "{definitions.note_ppp}"
+            hasMapTab: "{definitions.hasmaptab}"
+            tab: "{definitions.tab}"
+            hideRelativeToggle: false
+            yAxis:
+              min: 0
+              canChangeScaleType: true
+            map:
+              colorScale:
+                baseColorScheme: "Greens"
+                binningStrategy: "manual"
+                customNumericValues: *map_brackets_avg
+            selectedEntityNames: *entity_names_incomes_across_distribution
+
+      share:
+        title: Share ({definitions.percentiles_avg_share}, <<ppp_version>> prices) - <<table>> (<<welfare_type>>){definitions.survey_comparability}
+        unit: "%"
+        short_unit: "%"
+        description_short: The share of <<welfare_type>> {definitions.welfare_type_verb} by the {definitions.percentiles_avg_share} (tenth of the population)
+        description_key:
+          - "{definitions.description_key_welfare_type}"
+          - "{definitions.description_key_not_market_income}"
+        description_processing: |-
+          {definitions.description_processing_main}
+        display:
+          name: Share ({definitions.percentiles_avg_share})
+          numDecimalPlaces: 1
+          <<: *common_display
+        presentation:
+          grapher_config:
+            title: <<welfare_type.capitalize()>> share of the {definitions.percentiles_avg_share}
+            subtitle: The share of <<welfare_type>> {definitions.welfare_type_verb} by the {definitions.percentiles_avg_share} (tenth of the population).
+            note: "{definitions.note_without_ppp}"
+            hasMapTab: "{definitions.hasmaptab}"
+            tab: "{definitions.tab}"
+            yAxis:
+              min: 0
+            map:
+              colorScale:
+                baseColorScheme: "OrRd"
+                binningStrategy: "manual"
+                customNumericValues: *map_brackets_share
+                customNumericMinValue: "{definitions.custom_numeric_min_value_share}"
```

Comment:
@Marigold So all was fixed, except for this one! I am keeping this as zero for now, but I want to put "{definitions.custom_numeric_min_value_share}"

Reply:
Should be fixed.

Reply:
Thank you so much again!